src/runtime_src/core/common/info_aie.cpp

```diff
@@ -420,14 +420,16 @@ populate_aie_shim(const xrt_core::device *device, const std::string& desc)
         ....
 */
 
-// This funtion populate a specific AIE core given as an input of [row:col]
+// This function populate a specific AIE core given as an input of [row:col]
```

Comment:
Grammatically incorrect.   Simplify maybe:
```suggestion
// Populate a specific AIE core given as an input of [row:col]
```

Reply:
Fixed. Thanks